centreon/src/Core/AgentConfiguration/Infrastructure/Voters/AgentConfigurationVoter.php

```diff
@@ -41,6 +41,7 @@ final class AgentConfigurationVoter extends Voter
     use LoggerTrait;
     public const READ_AC = 'read_agent_configuration';
     public const READ_AC_POLLERS = 'read_agent_configuration_pollers';
+    public const READ_AVAILABLE_POLLERS = 'read_agent_configuration_available_pollers';
```

Comment:
Same logic could have been kept here no ?
```suggestion
    public const REAC_AC_AVAILABLE_POLLERS = 'read_agent_configuration_available_pollers';
```

Reply:
indeed